README.md

```diff
@@ -296,18 +310,19 @@ status code, the secret is considered verified.
 
 ```yaml
 # config.yaml
-detectors:
-- name: hog detector
-  keywords:
-  - hog
-  regex:
-    adjective: hogs are (\S+)
-  verify:
-  - endpoint: http://localhost:8000/
-    # unsafe must be set if the endpoint is HTTP
-    unsafe: true
-    headers:
-    - 'Authorization: super secret authorization header'
+customDetectors:
+  detectors:
```

Comment:
The reason I went this way was because in order to keep `detectors` top level, we'd need to add the new detector configs to `custom_detectorspb.CustomDetectors`. It seemed like it could cause confusion to add non-custom detector config to the custom detector proto. I made sure the original format work too so that existing configs will continue to work.

Reply:
I might just be overlooking something, but I haven't found a good, clean way to keep detectors top-level (other than the parsing both formats for compatibility).